<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPANSOPYSIDInitialDockWidget</class>
 <widget class="QDockWidget" name="QPANSOPYSIDInitialDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SID Initial Climb</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SID Initial Climb</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="layerGroupBox">
      <property name="title">
       <string>Input Layer</string>
      </property>
      <layout class="QFormLayout" name="layerFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="runwayLayerLabel">
         <property name="text">
          <string>Runway/DER:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsMapLayerComboBox" name="runwayLayerComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="elevationGroupBox">
      <property name="title">
       <string>Elevations</string>
      </property>
      <layout class="QFormLayout" name="elevationFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="adElevLabel">
         <property name="text">
          <string>Aerodrome Elev (m):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="adElevSpinBox">
         <property name="minimum">
          <double>-500.000000000000000</double>
         </property>
         <property name="maximum">
          <double>5000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="derElevLabel">
         <property name="text">
          <string>DER Elevation (m):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="derElevSpinBox">
         <property name="minimum">
          <double>-500.000000000000000</double>
         </property>
         <property name="maximum">
          <double>5000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="parametersGroupBox">
      <property name="title">
       <string>Flight Parameters</string>
      </property>
      <layout class="QFormLayout" name="parametersFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="pdgLabel">
         <property name="text">
          <string>PDG (%):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="pdgSpinBox">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>25.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>3.300000000000000</double>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="tempLabel">
         <property name="text">
          <string>Ref. Temp (°C):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="tempSpinBox">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>60.000000000000000</double>
         </property>
         <property name="value">
          <double>15.000000000000000</double>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="iasLabel">
         <property name="text">
          <string>IAS (kt):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="iasSpinBox">
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>205</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="altitudeLabel">
         <property name="text">
          <string>Turn Altitude (ft):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="altitudeSpinBox">
         <property name="minimum">
          <double>100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>5000.000000000000000</double>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="bankAngleLabel">
         <property name="text">
          <string>Bank Angle (°):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="bankAngleSpinBox">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="windLabel">
         <property name="text">
          <string>Wind (kt):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="windSpinBox">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="pilotTimeLabel">
         <property name="text">
          <string>Pilot Time (s):</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="pilotTimeSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>11</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="directionGroupBox">
      <property name="title">
       <string>Direction</string>
      </property>
      <layout class="QVBoxLayout" name="directionLayout">
       <item>
        <widget class="QPushButton" name="directionButton">
         <property name="text">
          <string>Direction: Start → End</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logGroupBox">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="logLayout">
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="maximumHeight">
          <number>100</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
